### PR TITLE
Updated German Translation (again... but bigger)

### PR DESCRIPTION
**Pull request by Yoru-Kitsune**
If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [x] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [x] I have ensured that my code compiles, if applicable.
- [x] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

File: core/assets/bundles/bundle_de.properties [hunk 1]
```diff
@@ -80,31 +80,31 @@ schematic.texttag = Text-Tag
...
+schematic.tagdelconfirm = Diesen Tag wirklich löschen?
+schematic.tagexists = Es existiert bereits ein Tag mit diesem Namen.
 
 stats = Statistiken
 stats.wave = Wellen besiegt
-stats.unitsCreated = Gebaute Einheiten
-stats.enemiesDestroyed = Getötete Gegner
-stats.built = Gebäude gebaut
+stats.unitsCreated = Einheiten erschaffen
+stats.enemiesDestroyed = Gegner zerstört
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:89]: `Einheiten zerstört` would be better, but also the previous one was correct

File: core/assets/bundles/bundle_de.properties [hunk 2]
```diff
@@ -1,40 +1,40 @@
-credits.text = Entwickelt von [royal]Anuken[] - [sky]anukendev@gmail.com[]\n\n[gray]
+credits.text = Ein Spiel von [royal]Anuken[] - [sky]anukendev@gmail.com[]\n\n[gray]
 credits = Mitwirkende
 contributors = Übersetzer und Mitwirkende
-discord = Tritt dem Mindustry-Discord bei!
-link.discord.description = Der offizielle Mindustry Discord-Server
-link.reddit.description = Mindustry-Subreddit
-link.github.description = Quellcode des Spiels
+discord = Trette dem Mindustry Discord bei!
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:4]: `Tritt` was correct.

File: core/assets/bundles/bundle_de.properties [hunk 3]
```diff
@@ -80,31 +80,31 @@ schematic.texttag = Text-Tag
...
-schematic.tagexists = Dieses Tag gibt es schon.
+schematic.tagdelconfirm = Diesen Tag wirklich löschen?
+schematic.tagexists = Es existiert bereits ein Tag mit diesem Namen.
 
 stats = Statistiken
 stats.wave = Wellen besiegt
-stats.unitsCreated = Gebaute Einheiten
-stats.enemiesDestroyed = Getötete Gegner
-stats.built = Gebäude gebaut
+stats.unitsCreated = Einheiten erschaffen
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:88]: wrong, previous one was correct

File: core/assets/bundles/bundle_de.properties [hunk 4]
```diff
@@ -125,55 +125,55 @@ preparingconfig = Konfiguration vorbereiten
 preparingcontent = Inhalt vorbereiten
 uploadingcontent = Inhalt hochladen
 uploadingpreviewfile = Vorschaudatei hochladen
-committingchanges = Veränderungen werden übernommen
+committingchanges = Änderungen werden übernommen
 done = Fertig
-feature.unsupported = Dein System unterstützt dieses Feature nicht.
+feature.unsupported = Dein System unterstützt diese Funktion nicht.
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:130]: wrong, previous one was correct

File: core/assets/bundles/bundle_de.properties [hunk 5]
```diff
@@ -125,55 +125,55 @@ preparingconfig = Konfiguration vorbereiten
...
 
-mods.initfailed = [red]⚠[] Die vorherige Mindustry-Instanz konnte nicht starten. Dies lag wahrscheinlich an fehlerhaften Mods.\n\nDamit das Spiel starten kann, [red]wurden alle Mods deaktiviert.[]\n\nWenn du nicht willst, dass das passiert, kannst du es unter [accent]Einstellungen->Spiel->Mods bei Absturz deaktivieren[] ändern.
+mods.initfailed = [red]⚠[] Die vorherige Mindustry Sitzung konnte nicht starten. Möglicherweise lag dies an fehlerhaften Mods.\n\nUm das Spiel zu starten, [red]wurden alle Mods deaktiviert.[]\n\nDie können dies unter [accent]Einstellungen -> Spiel -> Mods bei Absturz deaktivieren[] ändern.
 mods = Mods
-mods.none = [lightgray]Keine Mods gefunden!
-mods.guide = Modding-Anleitung
+mods.none = [lightgray]Keine Mods wurden gefunden!
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:134]: should be `Es wurden keine Mods gefunden`, also previous one was correct

**Review by Shawak — CHANGES_REQUESTED** [review]
This is a pretty bad merge request. There are some improvements but also a lot of changes which are making sentences which were correct before incorrect in grammar or using a prett bad wording.

Wouldn't merge this as it is

**Comment by Anuken** [comment]
Seems dead?

**Pull request closed without merging**